glide.yaml

```diff
@@ -1,3 +1,8 @@
 package: github.com/wantedly/developers-account-mapper
 import:
 - package: github.com/mitchellh/cli
+- package: github.com/aws/aws-sdk-go
+  subpackages:
+  - aws
+  - aws/session
+  - aws/service/dynamodb
```

Comment:
@dtan4 Running `glide update` lists dozens of lines of subpackages in glied.yaml, while [glide.lock of slack-mention-converter](https://github.com/wantedly/slack-mention-converter/blob/master/glide.lock) is so clean
Is this what is should be? Am I doing anything wrong?